### DISCUSSION TITLE
Has anyone got supabase running on .net 8 webapi?

### DISCUSSION THREAD

**Discussion by wgrs** [supabase-community/supabase-csharp#134] [General]
I am struggling with getting supabase to run on a new .net 8 webapi project - I followed Milan's example but its throwing an error:  Unhandled exception. System.TypeLoadException: Could not load type 'Supabase.Client' from assembly 'Supabase, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'

There error is thrown immediately on dotnet run.

**Comment by wiverson** [comment]
This sort of error is almost always because of one of two things:

- You a missing a library
- You have a conflicting library (for example, a .NET or platform provided lib conflicts with a library added to the project)

Depending on how you installed the library you may wind up with either of these two scenarios. Resolving usually boils down to your project and/or IDE configuration.

The big hint here is the Supabase.Client missing means that for whatever reason presumably none of the libs are being loaded. FWIW I mostly use Unity and Godot on macOS with Rider. If you are, say, using VS on Windows you might try checking one of those forums...?

**Comment by wgrs** [comment]
I recreated the project and its working now
